code/modules/scripting/Scanner/Scanner.dm

```diff
@@ -118,13 +118,14 @@
 		for(, src.codepos<=lentext(code), src.codepos++)
 
 			var/char=copytext(code, codepos, codepos+1)
+			var/nextchar=copytext(code, codepos+1, codepos+2)

```

Comment:
If it's the last character on the string it generates a compile warning in the NTSL IDE:
```
>28: Unexpected token: /
```
It seems to not have any issues there.